### PR TITLE
sei: use ECS definition of geo.location

### PR DESCRIPTION
**Pull request by efd6**
## What does this PR do?

This performs the the following replace:

from:

	- description: Longitude and latitude\.
	  level: core
	  (name: .*\.geo\.location)
	  type: geo_point

or:

	- (name: .*\.geo\.location)
	  description: Longitude and latitude\.
	  example: '{ "lon": -73\.614830, "lat": 45\.505918 }'
	  type: geo_point

or:

	- description: Longitude and latitude\.
	  example: '{ "lon": -73\.614830, "lat": 45\.505918 }'
	  (name: .*\.geo\.location)
	  type: geo_point

to:

	- external: ecs
	  $1

over all packages owned by Security External Integrations, except for deprecated packages.

Changelog and manifest files were updated manually.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/akamai/data_stream/siem/fields/ecs.yml [hunk 1]
```diff
@@ -17,9 +17,7 @@
 - name: client.geo.region_iso_code
   external: ecs
 - name: client.geo.location
-  description: Longitude and latitude.
-  example: '{ "lon": -73.614830, "lat": 45.505918 }'
-  type: geo_point
+  external: ecs
```
Comment (kcreddy) [inline packages/akamai/data_stream/siem/fields/ecs.yml:20]: There's also a `source.geo.location` field in this file. Does it also need to be changed?

File: packages/cloudflare/data_stream/logpull/fields/ecs.yml [hunk 2]
```diff
@@ -17,9 +17,7 @@
 - name: client.geo.region_iso_code
   external: ecs
 - name: client.geo.location
-  description: Longitude and latitude.
-  example: '{ "lon": -73.614830, "lat": 45.505918 }'
-  type: geo_point
+  external: ecs
```
Comment (kcreddy) [inline packages/cloudflare/data_stream/logpull/fields/ecs.yml:20]: There are also other fields in this file namely `destination.geo.location` , `source.geo.location`, and `observer.geo.location`

**Review by djptek — APPROVED** [review]
changes look OK and versions match up

```
for p in `ls packages` ; do cv=`grep -- "- version" packages/${p}/changelog.yml | head -1 | sed -e "s/- //" -e "s/\"//g"` ; mv=`grep -- "^version:" packages/${p}/manifest.yml | sed -e "s/\"//g"` ; if [[ ${cv} = ${mv} ]]; then echo "${p} ok"; else echo "${p} version mismatch ${cv} != ${mv}"; fi ; done
```

LGTM

File: packages/pulse_connect_secure/data_stream/log/fields/ecs.yml [hunk 3]
```diff
@@ -100,10 +96,8 @@
   name: source.geo.country_iso_code
 - external: ecs
   name: source.geo.country_name
-- description: Longitude and latitude.
-  level: core
+- external: ecs
```
Comment (kcreddy) [inline packages/pulse_connect_secure/data_stream/log/fields/ecs.yml:99]: `source.geo.location` is duplicated

File: packages/zscaler_zia/changelog.yml [hunk 4]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.4.2"
```
Comment (kcreddy) [inline packages/zscaler_zia/changelog.yml:2]: This doesn't have ECS fields of that type. Can be ignored?

**Review by kcreddy — APPROVED** [review]
Added some comments, but approved it.
These are the packages ( -> data streams) which seem to be missing the same updates:

```
atlassian_bitbucket
cloudflare -> audit
google_workspace -> groups, login, saml
hashicorp_vault
keycloak
mattermost
netskope
panw_cortex_xdr
slack
sophos
zscaler_zpa -> app_connector_status
zscaler_zpa -> browser_access
```

**Review by endorama — APPROVED** [review]
`gcp` package LGTM

File: packages/cloudflare/data_stream/logpull/fields/ecs.yml (hunk 2 above)
Comment (efd6) [reply packages/cloudflare/data_stream/logpull/fields/ecs.yml:20]: Also in audit.

**Comment by efd6** [comment]
- [x] atlassian_bitbucket
- [x] cloudflare -> audit
- [x] google_workspace -> groups, login, saml, *user_acounts*
- [ ] ~~hashicorp_vault~~ already using external definition
- [x] keycloak
- [x] mattermost
- [x] netskope
- [x] panw_cortex_xdr
- [x] slack
- [x] sophos
- [x] zscaler_zpa -> app_connector_status
- [ ] ~~zscaler_zpa -> browser_access~~ already using external definition

**Pull request merged**